plugins/autofrontlight.koplugin/main.lua

```diff
@@ -4,6 +4,10 @@ if not (Device:isKindle() and Device:hasLightSensor()) then
     return { disabled = true, }
 end
 
+if not Device:hasFrontlight() then
+    return { disabled = true, }
+end
+
```

Comment:
this one seems unnecessary, there is the `not (Device:isKindle() and Device:hasLightSensor())` check just above it and every kindle with light sensor already has a front light... and is being deprecated and removed soon nonetheless.

Reply:
Do you know why I still see it in the list?

Or, am I confusing it thinking that "disabled" should remove it from the list?